var/spack/repos/builtin/packages/py-bbp-workflow/package.py

```diff
@@ -12,28 +12,28 @@ class PyBbpWorkflow(PythonPackage):
     homepage = 'https://bbpcode.epfl.ch/code/#/admin/projects/nse/bbp-workflow'
     git      = 'ssh://bbpcode.epfl.ch/nse/bbp-workflow'
 
-    version('2.1.8', tag='bbp-workflow-v2.1.8')
+    version('2.1.11', tag='bbp-workflow-v2.1.11')
 
     depends_on('py-setuptools', type=('build'))
 
-    depends_on('py-luigi', type='run')
-    depends_on('py-luigi-tools', type='run')
-    depends_on('py-entity-management', type='run')
+    depends_on('py-luigi',               type='run')
+    depends_on('py-luigi-tools',         type='run')
+    depends_on('py-entity-management',   type='run')
     depends_on('py-requests-unixsocket', type='run')
-    depends_on('py-sh', type='run')
-    depends_on('py-matplotlib', type='run')
-    depends_on('py-dask+diagnostics', type='run')
-    depends_on('py-distributed', type='run')
-    depends_on('py-xarray', type='run')
-    depends_on('py-zarr', type='run')
-    depends_on('py-notebook', type='run')
-    depends_on('py-ipyparallel', type='run')
+    depends_on('py-sh',                  type='run')
+    depends_on('py-matplotlib',          type='run')
+    depends_on('py-dask+diagnostics',    type='run')
+    depends_on('py-distributed',         type='run')
+    depends_on('py-xarray',              type='run')
+    depends_on('py-zarr',                type='run')
+    depends_on('py-notebook',            type='run')
+    depends_on('py-ipyparallel',         type='run')
 
     depends_on('py-docutils', type='run')  # rdflib plugins pull this from python-daemon
 
-    depends_on('py-bluepy', type='run')
+    depends_on('py-bluepy',            type='run')
     depends_on('py-bluepy-configfile', type='run')
-    depends_on('py-simwriter', type='run')
+    depends_on('py-simwriter',         type='run')
```

Comment:
In `setup.py` there is also a dependency for `voxcell`. Shouldn't it be added?

Reply:
Done.